NewFile.jsp

```diff
@@ -189,7 +189,7 @@ footer
 <sql:setDataSource user="root" password="Fathihachellam.1" url="jdbc:mysql://localhost:3306/ooad?autoReconnect=true&&useSSL=false" driver="com.mysql.jdbc.Driver" var="db"/>

```

Comment:
@201831990711 

Probably it is not a good idea to show database password in a public repo.

-Hui